tests/fake_apns_server.py

```diff
@@ -203,16 +207,15 @@ def create_protocol(self):
 
 
 def main():
-    loop = asyncio.get_event_loop()
-    server = loop.run_until_complete(start_fake_apns_server())
-    device_id = server.create_device()
-    print(f'Serving on {server.address}')
-    print(f'Device ID: {device_id}')
-    try:
-        loop.run_forever()
-    except KeyboardInterrupt:
-        print('Stopping')
-        loop.run_until_complete(server.stop())
+    async def helper():
+        async with start_fake_apns_server() as server:
+            device_id = server.create_device()
+            print(f'Serving on {server.address}')
+            print(f'Device ID: {device_id}')
+            while True:
+                await asyncio.sleep(1)
```

Comment:
Is there a way around this? Maybe this would work?

```python
try:
    await asyncio.Future()
except KeyboardInterrupt:
    return
```

Reply:
Done.
I don't finish outstanding coros, but I figure that's OK for the test tool :)